packages/libs/src/sdk/api/albums/types.ts

```diff
@@ -30,6 +30,7 @@ export const createUploadAlbumMetadataSchema = () =>
       releaseDate: z.optional(
         z.date().max(new Date(), { message: 'should not be in the future' })
       ),
+      ddexReleaseIds: z.optional(z.record(z.string()).nullable()),
```

Comment:
nit: might be good if this (and elsewhere) had the full type like how you defined it in golang, ie:
```go
type ReleaseIDs struct {
	PartyID       string `bson:"party_id,omitempty"`
	CatalogNumber string `bson:"catalog_number,omitempty"`
	ICPN          string `bson:"icpn,omitempty"`
	GRid          string `bson:"grid,omitempty"`
	ISAN          string `bson:"isan,omitempty"`
	ISBN          string `bson:"isbn,omitempty"`
	ISMN          string `bson:"ismn,omitempty"`
	ISRC          string `bson:"isrc,omitempty"`
	ISSN          string `bson:"issn,omitempty"`
	ISTC          string `bson:"istc,omitempty"`
	ISWC          string `bson:"iswc,omitempty"`
	MWLI          string `bson:"mwli,omitempty"`
	SICI          string `bson:"sici,omitempty"`
	ProprietaryID string `bson:"proprietary_id,omitempty"`
}
```


but it's not a blocker

Reply:
yeah was just thinking we might adjust this as we develop the parser and didn't want to have to edit the type in 50 different places like is currently the case with track / playlist metadata... would be down to add it later after seeing more examples.